### PR TITLE
Csproj files cleanup

### PR DESCRIPTION
**Pull request by AndreyTretyak**
* Remove DevelopmentDependency where it's not needed
* Move common properties to Directories.Build.props

### CODE REVIEW

File: Directory.Build.props [hunk 1]
```diff
@@ -8,45 +8,50 @@
...
     <TargetPlatform>AnyCPU</TargetPlatform>
-    <PackageOutputPath>$(MSBuildThisFileDirectory)\nuget</PackageOutputPath>
-    <DocumentationFileValue>$(MSBuildThisFileDirectory)\bin\$(MSBuildProjectName)\$(Configuration)\$(TargetFramework)\$(MSBuildProjectName).xml</DocumentationFileValue>    
+    <ErrorReport>prompt</ErrorReport>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <WarningsAsErrors />
+    <CheckForOverflowUnderflow>true</CheckForOverflowUnderflow>
+    <AllowUnsafeBlocks>false</AllowUnsafeBlocks>
   </PropertyGroup>
   <PropertyGroup>
```
Comment (vlad-ion) [inline Directory.Build.props]: you might want to label this group and the one above also

Comment (AndreyTretyak) [reply Directory.Build.props]: I was not able to come up with nice names for now :(

Comment (AndreyTretyak) [reply Directory.Build.props]: Updated, thanks

Comment (vlad-ion) [reply Directory.Build.props]: Perfect.

**Pull request merged**